beacon_node/beacon_chain/src/validator_custody.rs

```diff
@@ -217,7 +217,7 @@ impl CustodyContext {
                     new_custody_group_count: updated_cgc,
                     sampling_count: self
                         .num_of_custody_groups_to_sample(Some(effective_epoch), spec),
-                    slot: current_slot,
+                    effective_epoch: current_slot.epoch(E::slots_per_epoch()),
```

Comment:
Sorry i was probably unclear again, what i meant was to use the existing `effective_eppch` variable in scope:

```suggestion
                    effective_epoch,
```

Reply:
[dc04308](https://github.com/sigp/lighthouse/pull/7648/commits/dc043083375a855b4fe8ec5aeaa05a5539539860)